Mathlib/MeasureTheory/Function/L1Space.lean

```diff
@@ -439,11 +439,11 @@ theorem integrable_const [IsFiniteMeasure μ] (c : β) : Integrable (fun _ : α
 
 @[simp]
 theorem Integrable.of_finite [Finite α] [MeasurableSpace α] [MeasurableSingletonClass α]
-    (μ : Measure α) [IsFiniteMeasure μ] (f : α → β) : Integrable (fun a ↦ f a) μ :=
+    {μ : Measure α} [IsFiniteMeasure μ] {f : α → β} : Integrable (fun a ↦ f a) μ :=
   ⟨(StronglyMeasurable.of_finite f).aestronglyMeasurable, .of_finite⟩
 
-lemma Integrable.of_isEmpty [IsEmpty α] (f : α → β) (μ : Measure α) :
-    Integrable f μ := Integrable.of_finite μ f
+@[deprecated (since := "2024-10-01")]
```

Comment:
Please add the lemma this is deprecated in favour of.

Reply:
This lemma was added with the knowledge that it was a special case of another one. The point was to help discoverability. It's not a simp lemma or anything like that, so I guess it does not hurt to have it?

Reply:
Oh sorry. But `stronglyMeasurable_of_isEmpty` already is deprecated?

Reply:
Yes, I see that! That's a clear sign that my opinion is not shared by everyone.

Thinking about it, I actually like having something on the lemma that tells the user that they should use a more general result, but the issue with deprecating is that someone will delete the lemma entirely in a few months. Ideally I'd like a permanent deprecated lemma.

Reply:
Would a docstring on the lemma saying this work?

Reply:
I would be happy with a docstring instead of deprecating.

Reply:
What about deprecating without `(since := )` + a docstring? See my last commit